packages/eas-cli/src/project/publish.ts

```diff
@@ -140,11 +143,60 @@ export async function buildUpdateInfoGroupAsync(assets: CollectedAssets): Promis
   return updateInfoGroup as UpdateInfoGroup;
 }
 
+export async function buildBundlesAsync({
+  projectDir,
+  inputDir,
+}: {
+  projectDir: string;
+  inputDir: string;
+}) {
+  const packageJSON = await readJSON(path.resolve(projectDir, 'package.json'));
```

Comment:
```suggestion
  const packageJSON = JsonFile.read(path.resolve(projectDir, 'package.json'));
```
Already imported in this file.